### PR TITLE
SDK - Failing faster in python_op tests

### PR DESCRIPTION
**Pull request by Ark-kun**
Added process error code check.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1291)

### CODE REVIEW

File: sdk/python/tests/components/test_python_op.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def helper_test_2_in_1_out_component_using_local_call(self, func, op):
                 task = op(arg1, arg2)
 
             full_command = task.command + task.arguments
-            process = subprocess.run(full_command)
+            subprocess.check_call(full_command)
```
Comment (gaoning777) [inline sdk/python/tests/components/test_python_op.py]: Why not set the "check" in subprocess.run call(https://docs.python.org/3/library/subprocess.html#subprocess.run).

subprocess,check_call is one of the old API(https://docs.python.org/3/library/subprocess.html#older-high-level-api), which i do not think should be used extensively.

Comment (Ark-kun) [reply sdk/python/tests/components/test_python_op.py]: Thanks for the insight.
I've switched back to `subprocess.run`.

**Pull request merged**